openml-lightgbm/lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/SWIGResources.java

```diff
@@ -290,14 +290,14 @@ public void close() {
     public int getBoosterNumIterations() { return this.boosterNumIterations; }
 
     /**
-     * Returns the number of features in the model.
+     * Returns the number of booster features in the model.
      *
      * @return boosterNumIterations
      */
     public int getBoosterNumFeatures() { return this.boosterNumFeatures; }
 
     /**
-     * Returns the name of the features in the model.
+     * Returns the name of booster features in the model.
```

Comment:
@shenggwang @jlazevedo the booster IS the model. I don't understand, it's like "Returns the name of model features in the model" in my head.

maybe "Returns the ... features in the LightGBM model" ?

Reply:
even that is a little pleonasmic :p